guides/Getting Started.md

```diff
@@ -0,0 +1,696 @@
+# Getting Started with Ecto
+
+This guide is an introduction to [Ecto](https://github.com/elixir-lang/ecto),
+the database wrapper and query generator for Elixir. Ecto provides a
+standardised API for talking to all the different kinds of databases, so that
+Elixir developers can query whatever database they're using in the same
+fashion. If one application uses MySQL and another uses PostgreSQL but both
+use Ecto, then the database querying for both of those applications will be
+almost identical.
+
+**If you want to see the code from this guide, you can view it [at ecto/examples/friends on GitHub](https://github.com/elixir-lang/ecto/tree/master/ecto/friends).**
+
+If you've come from the Ruby language, the "equivalent" there would be Active
+Record, Data Mapper, or Sequel. Java has Hibernate, and so on.
+
+In this guide, we're going to learn some basics about Ecto, such as creating,
+reading, updating and destroying records from a PostgreSQL database. 
+
+**This guide will require you to have setup PostgreSQL beforehand.**
+
+## Adding Ecto to an application
+
+To start off with, we'll generate a new Elixir application by running this command:
+
+```
+mix new friends --sup
+```
+
+The `--sup` option ensures that this application has [a supervision tree](http://elixir-lang.org/getting-started/mix-otp/supervisor-and-application.html), which we'll need for Ecto a little later on.
+
+To add Ecto to this application, there are a few steps that we need to take. The first step will be adding Ecto and an adapter called Postgrex to our `mix.exs` file, which we'll do by changing the `deps` definition in that file to this:
+
+```elixir
+defp deps do
+  [
+    {:ecto, "2.0.1"},
+    {:postgrex, "0.11.2"}
+  ]
+end
+```
+
+Ecto provides the common querying API, but we need the Postgrex adapter installed too, as that is what Ecto uses to speak in terms a PostgreSQL database can understand.
```

Comment:
It's a technicality, but `postgrex` is a driver, not adapter. The adapter is built into ecto.